### ISSUE TITLE
Real world accuracy of WCAG contrast formula?

### ISSUE THREAD

**Issue by incyphe** [w3c/wcag#928]
Hello,

I'm a UX practitioner and it's my first time posting :)

In general, I find that WCAG's calculated ratios do good job of indicating perceived contrast.  Looking under the hood, I see that it uses sRGB transformation formulas to accounts for non-linear ways in which people eyes perceive light, varying perceived luminescence of different colors, as well as ambient light factor.  All amazing stuff.

However, I can't help but notice one missing thing, which is, consideration for the fact that most people run their displays far brighter than sRGB recommended viewing spec of 80 cd/m2.

Most people have their displays at 250-300.  White point on 300 cd/m2 will look far brighter than the one calibrated to 80-100 cd/m2.

Possible consequence is, the formula underestimates real-life contrast of light on dark elements. In below screenshot, you'll see that 3:1 in the last column is far more legible than 3:1 or even 3.2:1 on other columns.

I notice that the effect goes down with dimmer LCD setting, but still present.

I know there are few other factors at play too including contrast setting, font choice, etc, but this is something I've observed over a few years across multiple devices.

What do you think?

![contrast-swatch](https://user-images.githubusercontent.com/3118215/68269462-e988c280-0027-11ea-8786-964ab46e3f4f.png)

Typeface replaced with rectangle blocks.
![contrast-swatch-2](https://user-images.githubusercontent.com/3118215/68269941-a3ccf980-0029-11ea-9d2f-0c59a478a9bf.png)

**Comment by JAWS-test** [comment]
Related: https://github.com/w3c/wcag/issues/360 and https://github.com/w3c/wcag/issues/695

**Comment by incyphe** [comment]
Thank you. It has overlap with #695 

I will close it.
